app/api/resources/user.py

```diff
@@ -60,9 +60,10 @@ def get(cls):
 
         page = request.args.get("page", default=UserDAO.DEFAULT_PAGE, type=int)
         per_page = request.args.get("per_page", default=UserDAO.DEFAULT_USERS_PER_PAGE, type=int)
+        is_verified = request.args.get("is_verified") == 'true'
 
         user_id = get_jwt_identity()
-        return DAO.list_users(user_id, request.args.get("search", ""), page, per_page)
+        return DAO.list_users(user_id, request.args.get("search", ""), page, per_page, is_verified = is_verified)
```

Comment:
do you need to specify this argument? Does it make sense to specify all other arguments in this call?